.github/workflows/_ci.yaml

```diff
@@ -129,10 +113,55 @@ jobs:
       BASE_LIBRARY: pax
     secrets: inherit
 
+  save-tags:
+    runs-on: ubuntu-22.04
+    needs: [build-jax, build-t5x, build-pallas, build-pax, build-rosetta-t5x, build-rosetta-pax]
+    outputs:
+      ALL_TAGS: ${{ steps.save-docker-tags.outputs.ALL_TAGS }}
+    steps:
+      - name: Save docker tags in a single JSON object
+        id: save-docker-tags
+        run: |
+          ALL_TAG_JSON=$(cat << EOF
```

Comment:
Need to update the output names once `_build.yaml`/`_build_rosetta.yaml` is updated.